src/explore-education-statistics-admin/src/components/editable/__tests__/ReorderableAccordion.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import noop from 'lodash/noop';
+import Button from '@common/components/Button';
+import ReorderableAccordion from '../ReorderableAccordion';
+
+describe('ReorderableAccordion', () => {
+  test("it doesn't show controls if `canUpdateRelease` is false", () => {
+    const canUpdateRelease = false;
+
+    render(
+      <ReorderableAccordion
+        canUpdateRelease={canUpdateRelease}
+        id="test"
+        onReorder={() => noop}
+        heading="test"
+        onSectionOpen={() => noop}
+        onToggleAll={() => noop}
```

Comment:
This isn't the best way of using `noop` We typically do this:

```tsx
onReorder={noop}
onSectionOpen={noop}
onToggleAll={noop}
```

Same with the other test case

Reply:
Cool good to know. I assumed that since you'd use `() => {}` to denote a function that returns void (if you're not using lodash) you'd do the same with lodash.